### DISCUSSION TITLE
フロント実装の進捗共有

### DISCUSSION THREAD

**Discussion by sakamo-bnn** [fLens-Team/fLens#15] [Meeting]
# 現状
※ 下記画像は物理的にカメラのレンズをカバーで隠している状態

![FireShot Capture 002 - fLens - flens-team github io](https://github.com/user-attachments/assets/60b05511-b08d-44cc-834b-4787502d3b4d)

- おおかたのUIは実装完了
- フィルタの変更
  - 現状プレビュー画像は全て「Normal」になってるが，クリックすると固有のフィルタに変更される
- 画像の保存

# 未実装部分
- 選択中の画像フィルタのフォーカス表示
- UIの調整（）
- 画面右上にあるタイポ「Media Art Project」をGithubのアイコンに変更
- フィルタのプレビュー画像

# お願いしたいこと
- 現状の評価
  - 保存した画像のファイル名を ```flens_日付_時刻_フィルタ名.png``` にする
- プレビュー画像の作成
- 引き継ぎ書

**Comment by daachan** [comment]
ファイル名を変更したい
2024-08-08_ファイル名　みたいな
